integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala

```diff
@@ -172,6 +173,13 @@ case class CreateTable(cm: TableModel) extends RunnableCommand {
 
     val tableInfo: TableInfo = TableNewProcessor(cm)
 
+    // Add validation for sort scope when create table
+    val sortScope = tableInfo.getFactTable.getTableProperties.get("sort_scope")
+    if (null != sortScope && !CarbonUtil.isValidSortOption(sortScope)) {
```

Comment:
`null != sortScope` can be removed, it is checked inside `CarbonUtil.isValidSortOption`